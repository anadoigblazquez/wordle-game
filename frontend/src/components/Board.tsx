import BoardField from "./BoardField"

const Board = () => {
  
  return (
    <div className="mb-[60px] max-w-fit mx-auto">
      <div className="
          row flex 
          gap-2 
          justify-between 
          mb-2">
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField /> 
      </div>
      <div className="
        row 
        flex 
        gap-2 
        justify-between 
        mb-2">
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField /> 
      </div>
      <div className="
        row 
        flex 
        gap-2 
        justify-between 
        mb-2">
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField /> 
      </div>
      <div className="
        row 
        flex 
        gap-2 
        justify-between 
        mb-2">
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField /> 
      </div>
      <div className="
        row 
        flex 
        gap-2 
        justify-between 
        mb-2">
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField />
        <BoardField /> 
      </div>
    </div>
  )
}

export default Board