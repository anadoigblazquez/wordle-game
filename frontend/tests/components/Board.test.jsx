import { render } from "@testing-library/react";
import Board from '../../src/components/Board'

describe( 'Tests on Board', () => {
    const className = 'className';

    test('should render Board', () => {
        const { container } = render( <Board className={ className } /> );
        expect( container ).toMatchSnapshot();
    })
    
})