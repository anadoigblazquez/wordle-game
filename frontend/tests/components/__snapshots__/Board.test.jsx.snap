// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on Board should render Board 1`] = `
<div>
  <div
    class="mb-[60px] max-w-fit mx-auto"
  >
    <div
      class=" row flex  gap-2  justify-between  mb-2"
    >
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
    </div>
    <div
      class=" row  flex  gap-2  justify-between  mb-2"
    >
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
    </div>
    <div
      class=" row  flex  gap-2  justify-between  mb-2"
    >
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
    </div>
    <div
      class=" row  flex  gap-2  justify-between  mb-2"
    >
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
    </div>
    <div
      class=" row  flex  gap-2  justify-between  mb-2"
    >
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
      <div
        class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
      >
        L
      </div>
    </div>
  </div>
</div>
`;
